src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1387,48 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+
+        // Get the row index corresponding to the com accelerometer sensor
+        // TODO: Correct the link index pssed here
+        IndexRange comAccelerometerRange = this->getRangeLinkSensorVariable(COM_ACCELEROMETER_SENSOR, 0);
+
+        for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_model.getNrOfLinks()); idx++)
+        {
+            // Get the column index corresponding to the net link external wrench sensor
+            IndexRange netExternalWrenchSensor = this->getRangeLinkSensorVariable(NET_EXT_WRENCH_SENSOR, idx);
+
+            iDynTree::Rotation base_R_m_link = base_H_m_links.at(idx).getRotation();
+
+//            Matrix3x3 base_R_m_link_eigen_matrix;
+
+//            base_R_m_link_eigen_matrix.setVal(0, 0, base_R_m_link.getVal(0,0));
+//            base_R_m_link_eigen_matrix.setVal(0, 1, base_R_m_link.getVal(0,1));
+//            base_R_m_link_eigen_matrix.setVal(0, 2, base_R_m_link.getVal(0,2));
+
+//            base_R_m_link_eigen_matrix.setVal(1, 0, base_R_m_link.getVal(1,0));
+//            base_R_m_link_eigen_matrix.setVal(1, 1, base_R_m_link.getVal(1,1));
+//            base_R_m_link_eigen_matrix.setVal(1, 2, base_R_m_link.getVal(2,2));
+
+//            base_R_m_link_eigen_matrix.setVal(2, 0, base_R_m_link.getVal(2,0));
+//            base_R_m_link_eigen_matrix.setVal(2, 1, base_R_m_link.getVal(2,1));
+//            base_R_m_link_eigen_matrix.setVal(2, 2, base_R_m_link.getVal(2,2));
+
+            // Get link to base rotation
+            matrixYElements.addSubMatrix(comAccelerometerRange.offset,
+                                         netExternalWrenchSensor.offset,
+                                         base_R_m_link);
+
+        }
+
+
+        // bY for the com acceleratio sensor is zeor
```

Comment:
the zero bias terms(and how?) also needs to be mentioned in the documentation.

Reply:
@prashanthr05 I updated the main issue with the details related to the measurements handling https://github.com/robotology/idyntree/issues/570#issuecomment-530347743
Please check it and see if any other additional information is needed